webpack.config.ts

```diff
@@ -11,28 +11,63 @@ import CopyWebpackPlugin from 'copy-webpack-plugin';
 import MiniCssExtractPlugin from 'mini-css-extract-plugin';
 import webpack, {Configuration} from 'webpack';
 
-function parseFeatureDetails(id: FeatureID): FeatureMeta {
-	const content = readFileSync(`source/features/${id}.tsx`, {encoding: 'utf-8'});
-	const fields = ['disabled', 'description', 'screenshot'] as const;
+const readmeContent = readFileSync('readme.md', 'utf-8');
+
+function stripLinks(markdownText: string): Partial<FeatureMeta> {
+	const urls: string[] = [];
+	const description = markdownText.replace(/\[(.+?)]\((.+?)\)/g, (_match, title, url) => {
+		urls.push(url);
+		return title;
+	});
+
+	return {
+		description,
+		screenshot: urls.find(url => url.startsWith('https://user-images.githubusercontent.com/')) ?? urls[0]
```

Comment:
How about this? Also no screenshot found should mean `screenshot: undefined`, not `url[0]`

```suggestion
		screenshot: urls.find(url => /\.(png|gif)$/.test(url))
```